src/libraries/Common/tests/System/Net/WebSockets/WebSocketStream.cs

```diff
@@ -274,7 +274,7 @@ private void ThrowIfDisposed()
                 ThrowObjectDisposedException();
             }
 
-            void ThrowObjectDisposedException() => throw new ObjectDisposedException(GetType().FullName);
+            void ThrowObjectDisposedException() => ObjectDisposedException.Throw(this);
```

Comment:
This method should be deleted, and `ObjectDisposedException.Throw(this)` should be called directly instead.